src/__tests__/__actions__/readArticleAction.test.js

```diff
@@ -0,0 +1,130 @@
+import moxios from "moxios";
+import dotenv from "dotenv";
+import configureMockStore from "redux-mock-store";
+import thunk from "redux-thunk";
+import axios from "../../utils/axios";
+import {
+  fetchingArticle,
+  fetchArticle,
+  deleteArticle
+} from "../../redux/actions/readArticleActionCreator";
+import {
+  FETCHING_ARTICLE,
+  FETCHING_ASIDE_ARTICLES,
+  ARTICLE_FETCHED,
+  ARTICLE_ERROR,
+  DELETE_ARTICLE
+} from "../../redux/actionTypes";
+
+dotenv.config();
```

Comment:
@YvesIraguha This should be configured globally to avoid importing it in every file